File: app/src/main/java/vn/com/basc/inco/gcm/Push.java
package vn.com.basc.inco.gcm;

import java.io.Serializable;

/**
 * Created by SONY on 6/19/2016.
 */
public class Push implements Serializable{
    private String title;
    private String photo;
    private String message;
    private String component;
    private String project;
    private String id;
    private String date;
    private String parent;
    private String attach;
    private long idDB;

    public String getParent() {
        return parent;
    }

    public String getAttach() {
        return attach;
    }

    public long getIdDB() {
        return idDB;
    }

    public void setParent(String parent) {
        this.parent = parent;
    }

    public void setIdDB(long idDB) {
        this.idDB = idDB;
    }

    public void setAttach(String attach) {
        this.attach = attach;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public void setPhoto(String photo) {
        this.photo = photo;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public void setComponent(String component) {
        this.component = component;
    }

    public void setProject(String project) {
        this.project = project;
    }

    public void setId(String id) {
        this.id = id;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getTitle() {
        return title;
    }

    public String getPhoto() {
        return photo;
    }

    public String getMessage() {
        return message;
    }

    public String getComponent() {
        return component;
    }

    public String getProject() {
        return project;
    }

    public String getId() {
        return id;
    }

    public String getDate() {
        return date;
    }
}

File: core/lib/gcm/push.php
<?php
/**
 * @author Ravi Tamada
 * @link URL Tutorial link
 */

class Push{
    // push message title
    private $title;
    private $photo;
    private $message;
    private $component; 
    private $project;
    private $id;
    private $date;
    private $parent;
    private $attach;
    function __construct() {
        
    }
     public function setDate($date){
        $this->date = $date;
    }
    public function setAttach($attach){
        $this->attach = $attach;
    }
    public function setParent($parent){
        $this->parent = $parent;
    }    
    public function setTitle($title){
        $this->title = $title;
    }
    
    public function setPhoto($photo){
        $this->photo = $photo;
    }
    public function setMessage($message){
        $this->message = $message;
    }
    public function setComponent($component){
        $this->component = $component;
    }
    public function setProject($project){
        $this->project = $project;
    }
    public function setID($id){
        $this->id = $id;
    }
    public function getPush(){
        $res = array();
        $res['title'] = $this->title;
        $res['photo'] = $this->photo;
        $res['message'] = $this->message;
        $res['component'] = $this->component;
        $res['project'] = $this->project;
        $res['parent'] = $this->parent;
        $res['attach'] = $this->attach;
        $res['date'] = $this->date;
        $res['id'] = $this->id;
        return $res;
    }
}